scripts/scr_ui_manage/scr_ui_manage.gml

```diff
@@ -718,39 +718,47 @@ function scr_ui_manage() {
 	            }
 	            temp3=string(round(ma_health[sel]))+"% HP";temp4="";
 	            // Need abbreviations here
-            
-            
-            
+
+
+
 	            ma_ar="";ma_we1="";ma_we2="";ma_ge="";ma_mb="";ttt=0;
 	            ar_ar=0;ar_we1=0;ar_we2=0;ar_ge=0;ar_mb=0;
-            
+
+							if (ma_armor[sel]!=""){ma_ar=scr_wep_abbreviate(ma_armor[sel]);
+	                // if (string_count("*",ma_ar)>0){ar_ar=2;ma_ar=string_replace(ma_ar,"*","");}
+	                if (string_count("^",ma_armor[sel])>0){ar_ar=1;ma_ar=string_replace(ma_ar,"^","");}
+	                if (string_count("&",ma_armor[sel])>0){ar_ar=2;ma_ar="Artifact";}
+	            }
 	            if (ma_gear[sel]!=""){ma_ge=scr_wep_abbreviate(ma_gear[sel]);
 	                // if (string_count("*",ma_ge)>0){ar_ge=2;ma_ge=string_replace(ma_ge,"*","");}
-	                if (string_count("^",ma_ge)>0){ar_ge=1;ma_ge=string_replace(ma_ge,"^","");}
-	                if (string_count("&",ma_ge)>0){ar_ge=2;ma_ge="Artifact";}
-                
+	                if (string_count("^",ma_gear[sel])>0){ar_ge=1;ma_ge=string_replace(ma_ge,"^","");}
+	                if (string_count("&",ma_gear[sel])>0){ar_ge=2;ma_ge="Artifact";}
+	            }
+	            if (ma_mobi[sel]!=""){ma_mb=scr_wep_abbreviate(ma_mobi[sel]);
+	                // if (string_count("*",ma_mb)>0){ar_mb=2;ma_mb=string_replace(ma_mb,"*","");}
```

Comment:
same?

Reply:
As above, want them removing or leaving for potential documentation regarding Duke's original work?

Reply:
I recommend to remove them. We will still have them in git history, so we can always check Duke's old code.
But this is a small issue, your call.

Reply:
Duke's original comments removed and my own comments regarding marine-vehicle equipment mapping added.